packages/babel-preset-cozy-app/README.md

```diff
@@ -90,6 +90,32 @@ By default, this babel preset uses [`babel-plugin-transform-runtime`](https://ba
 }
 ```
 
+#### Preset options
+
+You can have control on the options passed to `babel-preset-env` and `babel-plugin-transform-runtime`:
+
+* `presetEnv` will be passed to `babel-preset-env`
+* `transformRuntime` will be passed to `babel-plugin-transform-runtime`
```

Comment:
`transformRuntime` is a plugin option which is different from "preset options" categorised here.
Also, we already have a `transformRegenerator` option above, so it will be deprecated? Shouldn't we remove the existed one or maybe merge the both? If we pass an object it will be used as the plugin options, if we pass booleans it will enable/disable with the default config. What do you think? It's just about having a clearer API and documentation